src/components/ReportActionItem/IssueCardMessage.tsx

```diff
@@ -16,11 +16,9 @@ import {isEmptyObject} from '@src/types/utils/EmptyObject';
 
 type IssueCardMessageProps = {
     action: OnyxEntry<ReportAction>;
-
-    policyID: string | undefined;
 };
 
-function IssueCardMessage({action, policyID}: IssueCardMessageProps) {
+function IssueCardMessage({action}: IssueCardMessageProps) {
```

Comment:
@shubham1206agra Can you actually fix the lint on this file using the `getOriginalMessage` its low risk, one line. I originally thought it was the useOnyx issue, but this is the only linter issue, so please let's update it here. Thanks!